### PR TITLE
#393: task timer display

### PR DESCRIPTION
**Pull request by JanBan**
#393

### CODE REVIEW

File: Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/child_app/view/task_list/TaskRecyclerViewAdapter.java [hunk 1]
```diff
@@ -17,29 +17,53 @@
...
+                taskDuration.setVisibility(View.INVISIBLE);
+            }
+            else if (task.getDurationTime() != null) {
+                actionPicture.setImageResource(R.drawable.timer);
+                taskDuration.setText(String.format("%ss", task.getDurationTime().toString()));
+                hasTimer = true;
+            }
+            else {
+                actionPicture.setVisibility(View.INVISIBLE);
+                taskDuration.setVisibility(View.INVISIBLE);
```
Comment (SuperJarek) [inline Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/child_app/view/task_list/TaskRecyclerViewAdapter.java:64]: Not sure, but I think that task has always default time for timer (3 sec).

File: Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/child_app/view/task_list/TaskRecyclerViewAdapter.java [hunk 2]
```diff
@@ -17,29 +17,53 @@
...
+            }
+        }
     }
 
     TaskRecyclerViewAdapter(List<TaskTemplate> tasks, TaskItemClickListener taskItemClickListener) {
```
Comment (SuperJarek) [inline Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/child_app/view/task_list/TaskRecyclerViewAdapter.java:69]: There are two ClickListeners for task item right now. For certain we need StepsClickListener to be passed here, not sure about about TimerClickListener. So I would recommend to refactor taskItemClickListener to StepsIconListener/StepsClickListener, to make code more clear.

File: Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/child_app/view/task_list/TaskRecyclerViewAdapter.java (hunk 1 above)
Comment (SuperJarek) [reply Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/child_app/view/task_list/TaskRecyclerViewAdapter.java:64]: I mean in original app.

**Review by SuperJarek — CHANGES_REQUESTED** [review]

File: Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/child_app/view/task_list/TaskRecyclerViewAdapter.java (hunk 1 above)
Comment (JanBan) [reply Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/child_app/view/task_list/TaskRecyclerViewAdapter.java:64]: Resolved, it has not

**Pull request merged**